lib/src/view/game/game_body.dart

```diff
@@ -100,9 +102,24 @@ class GameBody extends ConsumerWidget {
       case AsyncData(value: final gameState, isRefreshing: false):
         final youAre = gameState.game.youAre ?? Side.white;
 
+        AsyncValue<Crosstable>? crosstable;
+        // If playing against Stockfish, user is null
+        if (gameState.game.white.user != null && gameState.game.black.user != null) {
```

Comment:
Using a `final crosstable` would be better; The ternary operator reads quite well, as it is correctly indented by dart.